src/modals/SendModals/SendModalTransfer.tsx

```diff
@@ -254,7 +263,17 @@ const handleSend = async (client: Client, transactionData: TransferTxData, conte
         context.unsignedTransaction.unsignedTx
       )
 
-      // TODO: dispatch pending tx
+      store.dispatch(
+        transactionSent({
+          hash: data.txId,
+          fromAddress: fromAddress.hash,
+          toAddress,
+          timestamp: new Date().getTime(),
+          type: 'transfer',
+          lockTime: lockTime?.getTime(),
+          status: 'pending'
+        })
```

Comment:
You might notice the amount is missing, I've added it back in the next branch and fixed some issues with types.